ramls/orders.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 1.0
+title: "mod-orders-storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v1
+
+documentation:
+  - title: mod-orders
+    content: <b>Get list of purchase orders API</b>
+
+types:
+    purchase_order: !include acq-models/mod-orders-storage/schemas/purchase_order.json
+    purchase_order_collection: !include acq-models/mod-orders-storage/schemas/purchase_order_collection.json
+
+traits:
+    orderable: !include raml-util/traits/orderable.raml
+    pageable:  !include raml-util/traits/pageable.raml
+    searchable: !include raml-util/traits/searchable.raml
+    language: !include raml-util/traits/language.raml
+
+resourceTypes:
+    collection-get: !include raml-util/rtypes/collection-get.raml
+
+/orders:
+  type:
+    collection-get:
+      exampleCollection: !include acq-models/mod-orders-storage/examples/purchase_order_collection.sample
+      exampleItem: !include acq-models/mod-orders-storage/examples/purchase_order_get.sample
```

Comment:
The `collection-get.raml` defines only `exampleCollection` and `schemaCollection` parameters